jobCreationProcess.js

```diff
@@ -0,0 +1,60 @@
+import { describe, it, expect } from 'vitest';
+
+describe("Successful job creation", () => {
+
+    //Sucessful job creation
+    it("Should return 200 OK with job details (ID, title, description, created date)", async () => {
+        const res = await request(app)
+            .get("/api/job")
+            .set(...(await gt()))
+            .send();
+
+        expect(res.status).toBe(200);
+        expect(res.body).toHaveProprty("jobId");
+        expect(res.body.title).toBe("title");
+        expect(res.description).toBe("description");
+        expect(res.body).toHaveProperty("createdDate");
+    });
+    
+    //Missing Required Fields
+    it("Should return 400 bad request with relevant error message", async () => {
+        const res = await request(app)
+            .get("/api/job")
+            .send({});
+
+        expect(res.status).toBe(400);
+        expecct(res.status).toBe("Required Field Missing");
+    });
+
+    //Invalid Data
+    it("Should return 400 Bad Request with a specific error message", async () => {
+        const res = await request(app)
+            .get("/api/job")
+            .send();
+
+        expect(res.status).toBe(400);
+        expect(res.status).toBe("Data not valid");
```

Comment:
Isn't status an instance of Number (as referenced on 35)? How could it be a String? You could be thinking of `statusText` which will always be the [accepted text associated with an error message](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status). Usually errors get passed back in an error message, accessible as `body.message`

The string you are searching for is not correct, but if it was this would look like: 

```suggestion
        expect(res.body.message).toBe("Data not valid");
```